### PR TITLE
begin rhv guides

### PR DESCRIPTION
**Pull request by arendej**
just like there's a vmware guide ;)

##### SUMMARY

starting a guide section for RHV/ oVirt just like there is one for VMware

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

RHV guide section in the docs.

##### ADDITIONAL INFORMATION

Just adding docs to match up content for RHV for what there is about VMware.

N/A

Note that this is an initial commit... more commits to come via this fork/branch,
If there's a better way to move this into the docs, please let me know.

### CODE REVIEW

**Comment by arendej** [comment]
will get to work on cleaning up the rst

File: docs/docsite/rst/rhv/faq.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+oVirt is the upsteam open-source edition of Red Hat Virtualization. 
+Red Hat Virtualization is an open-source alternative hypervisor to such like VMware. As of RHV version 4.1+ the feature-set and 
+maturity of RHV insists that it is prime for enterprise production workloads, among others.
+
+What version of oVirt / RHV will Ansible modules work with?
+------------------------------------------------------------
+
+Most of the more recently build ``ovirt_*`` modules are best experienced with oVirt/ RHV 4.1 or newer, hence the preview flags 
+on the modules, indicating no guarantee for backwards compatibility.
```
Comment (machacekondra) [inline docs/docsite/rst/rhv/faq.rst]: Note that we make the modules backward compatible, even when docs says it don't have to be.

File: docs/docsite/rst/rhv/rhv_concepts.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+The ordered lists of tasks, saved so you can run those tasks in that order
+repeatedly and practice infrastructure-as-code. Playbooks can include variables
+ as well as tasks. Playbooks are written in YAML and are easy to read, write, 
+ share and understand.
+
+
+ovirt-engine-sdk-python
+=======================
+
+Ansible RHV modules often require the ovirt-engine-sdk-python package installed.
```
Comment (machacekondra) [inline docs/docsite/rst/rhv/rhv_concepts.rst]: not often but always

**Review by machacekondra — COMMENTED** [review]
Nice, thank you for this. cc @mwperina

File: docs/docsite/rst/rhv/faq.rst (hunk 1 above)
Comment (arendej) [reply docs/docsite/rst/rhv/faq.rst]: how far back are they tested to work?

File: docs/docsite/rst/rhv/rhv_concepts.rst (hunk 2 above)
Comment (arendej) [reply docs/docsite/rst/rhv/rhv_concepts.rst]: will change, thanks.

File: docs/docsite/rst/rhv/faq.rst (hunk 1 above)
Comment (machacekondra) [reply docs/docsite/rst/rhv/faq.rst]: It should work on all 4.x versions. The only limitation is SDK, which must be always the latest.

**Comment by acozine** [comment]
@funixz thanks for working on this - please put the files in `docs/docsite/rst/scenario_guides/rhv` instead of directly under `rst` - we will be moving the vmware files into a similar directory

**Comment by acozine** [comment]
@funixz since we merged #54554, all platform-specific guides live in `docs/docsite/scenario_guides/*`. If you're still working on the RHV materials, please move the files into a new `docs/docsite/scenario_guides/rhv` directory.

**Comment by funixz** [comment]
will do, thanks!

**Comment by acozine** [comment]
@funixz now you have the files in both places on the branch - can you remove the ones in the `docs/docsite/rst/rhv` directory? You've also got some CI failure for things like underlining that is too short for the header, etc. - the most recent version of our [style guide](https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html) may help. Feel free to ask questions in the Ansible docs community chat on the #ansible-docs channel of freenode IRC.

**Comment by mattclay** [comment]
bot_status

**Comment by acozine** [comment]
@funixz I pushed a few commits to see if I can get this PR passing on Shippable. I steered you wrong about which directory to put the files in, so I corrected that (and also updated my comment for anyone else who finds this PR and uses it as a model for future work . . .) and updated the intra-page TOCs to match our [style guide](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#adding-local-tocs).

**Comment by acozine** [comment]
@funixz could you pick this PR back up again? I've got it organized correctly and passing CI, but I think it should be an `oVirt Guide` rather than a `RHEV Guide`. Can you do a search-and-replace mission on these pages?

**Comment by funixz** [comment]
> @funixz could you pick this PR back up again? I've got it organized correctly and passing CI, but I think it should be an `oVirt Guide` rather than a `RHEV Guide`. Can you do a search-and-replace mission on these pages?

can do.

**Comment by funixz** [comment]
dropped into rebase hell so I'll submit the changes to a new branch. close/delete this branch

**Comment by samccann** [comment]
Closing in favor of  #59413

**Pull request closed without merging**